lighthouse-core/report/html/report-styles.css

```diff
@@ -432,6 +432,12 @@
   display: none;
 }
 
+.lh-audit--informative .lh-audit__score-icon {
```

Comment:
since this PR doesn't render anything, seems like this should be in a different PR? (really it could be on it's own)